README.md

```diff
@@ -4,11 +4,11 @@
 
 ## Overview
 
-BTP Manager is an operator for [SAP BTP Service Operator](https://github.com/SAP/sap-btp-service-operator) based on the [Kubebuilder](https://github.com/kubernetes-sigs/kubebuilder) framework. It extends Kubernetes API by providing [BtpOperator](/config/crd/bases/operator.kyma-project.io_btpoperators.yaml) CRD which allows you to manage SAP BTP Service Operator resource through CR. 
+BTP Manager is an operator for the [SAP BTP service operator](https://github.com/SAP/sap-btp-service-operator) based on the [Kubebuilder](https://github.com/kubernetes-sigs/kubebuilder) framework. It extends Kubernetes API by providing [BtpOperator CustomResourceDefinition](/config/crd/bases/operator.kyma-project.io_btpoperators.yaml) (CRD) which allows you to manage the SAP BTP service operator resource through custom resource (CR). 
 
 ## Installation
 
-To enable the SAP BTP Operator module from the latest release, you must install BTP Manager and SAP BTP Service Operator.
+To enable the SAP BTP Operator module from the latest release, you must install BTP Manager and the SAP BTP service operator.
```

Comment:
I'm not sure I would add "the" if it's not absolutely necessary (see [SAP Styleguide](https://help.sap.com/docs/SNG_FOR_UA/e33c591ae4494a659a3f5f983c9d1161/60052027ac4b4b4eab7bb3bc24571640.html?locale=en-US&state=DRAFT#descriptors-and-articles)). It's not strictly forbidden, but "no article" is preferred over using one.

Reply:
I expected this comment :D So, I did research to be on the safe side. I found "the" before "SAP BTP service operator in HP inline documentation and GitHub." I guess I shouldn't make these docs an exception.